keyboards/kprepublic/cospad/readme.md

```diff
@@ -1,11 +1,12 @@
-COSPAD
-===
+#COSPAD
+
+![COSPAD](https://ae01.alicdn.com/kf/HTB1.lwARFXXXXcKXXXXq6xXFXXXK.jpg?size=289297&height=518&width=750&hash=7c5d826f928b91436cb36dca2746b424)
 
 A DIY Keypad Kit sold by KPRepublic, runs TKG natively.
 
 Keyboard Maintainer: QMK Community  
 Hardware Supported: COSPAD  
-Hardware Availability: [KPrepublic on Aliexpress](https://aliexpress.com/item/cospad-Custom-Mechanical-Keyboard-Kit-up-tp-24-keys-Supports-TKG-TOOLS-Underglow-RGB-PCB-20/32818383873.html)
+Hardware Availability: [KPrepublic](https://kprepublic.com/collections/cospad-xd24/products/cospad-20-numpad-xd24-xiudi-custom-keyboard-pcb)
```

Comment:
I wouldn't call this *wrong*, but the AliExpress link is still valid.

I'd prefer including both, and then anyone interested in this board can use the purchase link that works best for them.

```suggestion
Hardware Availability: [KPrepublic](https://kprepublic.com/collections/cospad-xd24/products/cospad-20-numpad-xd24-xiudi-custom-keyboard-pcb); [AliExpress](https://aliexpress.com/item/cospad-Custom-Mechanical-Keyboard-Kit-up-tp-24-keys-Supports-TKG-TOOLS-Underglow-RGB-PCB-20/32818383873.html)
```

Reply:
Good idea with including both, the main reason was one of the links was tinyurl and asked me to log in.

Reply:
Good spot on that. I tested the old link too and got the same thing.

Perhaps I should change my AliExpress password now... :eyes:

Reply:
I didn't get any interaction with LastPass when i visited the link, go i guess it should be ok. No idea if it had a referral link.